### ISSUE TITLE
Audio hiccup and high pitch

### ISSUE THREAD

**Issue by Inamaru** [Franiac/TwitchLeecher#630]
Hi, 
As title says, I'm having troubles with this stream. (PS: Only happens with this one at the moment, tried to download multiples times and still didn't work, never had any troubles with other streams (even from the same streamer channel.)

It seems to be working and downloading everything fine, but when I open it with VLC the audio is having hiccup + high pitch for the entire video (there are parts that are muted, but those are obviously for copyright infringement.)

I would like to point out that I download the source and don't select anything specific within the tool you provide.
I also used VLC (latest version) for viewing the content.
I'm on Windows 10 N home 64bit (Not sure if this helps, but I've got the media pack installed, even though VLC doesn't need it)

I'm not sure if this problem is on my side but thanks in advance for checking it out and sorry for the trouble.

<details>

<summary>TWITCH LEECHER INFO</summary>
--------------------------------------------------------------------------------------------
Version: 1.8.4

VOD INFO
--------------------------------------------------------------------------------------------
VOD ID: 315604464
Selected Quality: 1280x720@30fps (Source)
Download Url: https://www.twitch.tv/videos/315604464
Crop Start: No
Crop End: No

OUTPUT INFO
--------------------------------------------------------------------------------------------
Disable Conversion: No
Output File: C:\Users\Inamaru\Downloads\20180928_315604464_Art.mp4
FFMPEG Path: C:\Program Files\Twitch Leecher\ffmpeg.exe
Temporary Download Folder: C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955

ACCESS INFO
--------------------------------------------------------------------------------------------
Token: %7B%22authorization%22%3A%7B%22forbidden%22%3Afalse%2C%22reason%22%3A%22%22%7D%2C%22chansub%22%3A%7B%22restricted_bitrates%22%3A%5B%5D%7D%2C%22device_id%22%3Anull%2C%22expires%22%3A1628343362%2C%22https_required%22%3Atrue%2C%22privileged%22%3Afalse%2C%22user_id%22%3Anull%2C%22version%22%3A2%2C%22vod_id%22%3A315604464%7D
Signature: 7bbe7d73d9f1632d1f5bc57f49977549987ec42c
Sub-Only: No
Privileged: No

Creating temporary download directory 'C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955'... done!

Retrieving m3u8 playlist urls for all VOD qualities... done!
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/chunked/highlight-315604464-muted-P7GVKQFLNC.m3u8
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/720p30/highlight-315604464-muted-P7GVKQFLNC.m3u8
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/480p30/highlight-315604464-muted-P7GVKQFLNC.m3u8
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/audio_only/highlight-315604464-muted-P7GVKQFLNC.m3u8
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/360p30/highlight-315604464-muted-P7GVKQFLNC.m3u8
https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/160p30/highlight-315604464-muted-P7GVKQFLNC.m3u8

Playlist url for selected quality 1280x720@30fps (Source) is https://d2nvs31859zcd8.cloudfront.net/48a6c7a6934beae477f7_hozure_30538391920_977274926/chunked/highlight-315604464-muted-P7GVKQFLNC.m3u8

Retrieving playlist... done!
Parsing playlist... done!
Number of video chunks: 1901

Starting parallel video chunk download
Number of video chunks to download: 1901
Maximum connection count: 10

Parallel video chunk download is running...

Download of all video chunks complete!

Merging all VOD parts into 'C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955\20180928_315604464_Art.ts'...

Executing 'C:\Program Files\Twitch Leecher\ffmpeg.exe' on 'C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955\20180928_315604464_Art.ts'...
Command line arguments: -y -i "C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955\20180928_315604464_Art.ts" -analyzeduration 2147483647 -probesize 2147483647 -c:v copy -c:a copy "C:\Users\Inamaru\Downloads\20180928_315604464_Art.mp4"

Video conversion complete!

Starting temporary download folder cleanup!
Deleting directory 'C:\Users\Inamaru\AppData\Local\Temp\TL_779c8e2a-c1ec-4247-948c-a77e85112955'... done!

Download task ended successfully!

</details>

**Comment by Franiac** [comment]
There is definitely no issue after downloading the stream on my end. Everything works normal and audio is smooth. Seems like an issue on your side, sorry

**Comment by Inamaru** [comment]
Okay good to know, I'll try to look into possible solution, thanks again.

**Comment by Inamaru** [comment]
Hi, I'm sorry to bother again but, I tried to download the stream again on a brand new machine, and I'm still having the same issues,
I'm genuinely curious how you manage to get the streaming without any of the problems I pointed out earlier.

I mean, I'm having this problem ONLY with this stream, there has to be something that I'm doing wrong or something that's missing. 
The pitched audio is not that high, but if you compare the voices on the downloaded video vs the stream (especially in the end) they're on a totally different tone, and still, on the whole video every 3-4 secs the audio just freeze as usual. (the video playback is perfect though).
